active_model_serializers.gemspec

```diff
@@ -42,7 +42,7 @@ Gem::Specification.new do |spec|
   # 'minitest'
   # 'thread_safe'
 
-  spec.add_runtime_dependency 'jsonapi-renderer', ['>= 0.1.1.beta1', '< 0.2']
+  spec.add_runtime_dependency 'jsonapi-renderer', ['>= 0.1.1.beta1', '< 0.3']
```

Comment:
This is the same as `~> 0.2.0`, except it also allows the `0.1` series. Any specific reason?

Reply:
@beauby maybe I didn't get this comment: https://github.com/rails-api/active_model_serializers/pull/2189#pullrequestreview-63233013

in response to: https://github.com/rails-api/active_model_serializers/pull/2189#discussion_r139317044

Reply:
I'm not sure what @bf4 meant there either (:

Reply:
@beauby I thought it would be a breaking change to AMS to change the minimum jsonapi-renderer version.  Am I wrong there?

Reply:
Oh right – yeah then `['>= 0.1.1.beta1', '< 0.3']` LGTM.

Reply:
Should I do something else on my side?

Reply:
@taglia No, I think we're good. @bf4?